content/applications/finance/fiscal_localizations/spain.rst

```diff
@@ -2,34 +2,180 @@
 Spain
 =====
 
-Spanish chart of accounts
-=========================
+Configuration
+=============
 
-Several Spanish charts of accounts are available by default in Odoo:
+Install the 🇪🇸 **Spanish** :doc:`fiscal localization package <../fiscal_localizations>` to get all
+the default accounting features of the Spanish localization.
 
-- PGCE PYMEs 2008
-- PGCE Completo 2008
-- PGCE Entitades
+Chart of accounts
+=================
 
-To choose the one you want, go to :menuselection:`Accounting --> Configuration --> Settings`,
-then select a package in the :guilabel:`Fiscal localization` section.
+You can reach the **Chart of Accounts** by going to :menuselection:`Accounting --> Configuration -->
+Accounting: Chart of Accounts`.
+
+The **Spanish** chart of accounts includes pre-configured accounts by default:
+
+- PGCE PYMEs 2008;
+- PGCE Completo 2008;
+- PGCE Entitades sin animo de lucro 2008.
+
+To select the one to use, go to :menuselection:`Accounting --> Configuration --> Settings`, and
+select a package in the :guilabel:`Fiscal localization` section.
 
 .. tip::
     When you create a new Odoo Online database, the PGCE PYMEs 2008 is installed by default.
 
 .. warning::
    You can only change the accounting package as long as you have not created any accounting entry.
 
-Spanish accounting reports
-==========================
+Taxes
+=====
+
+Default Spanish taxes are created automatically when the
+:guilabel:`Spanish - Accounting (PGCE 2008)`, and the :guilabel:`Spanish - Accounting (PGCE 2008)`
+modules are installed.  Each tax impacts the Spanish **Tax Reports (Modelo)**, available by going to
+:menuselection:`Accounting --> Reporting --> Statements Reports: Tax Report`.
+
+Reports
+=======
+
+Here is the list of Spanish-specific reports available:
+
+- Balance Sheet;
+- Profit & Loss;
+- EC Sales List;
+- Tax Report (Modelo 111);
+- Tax Report (Modelo 115);
+- Tax Report (Modelo 303);
+- Tax Report (Modelo 347);
+- Tax Report (Modelo 349);
+- Tax Report (Modelo 390).
+
+You can access Spanish-specific versions of reports by clicking on the **book** icon when on a
+report and selecting its Spanish version: :guilabel:`(ES)`.
```

Comment:
There's only "one" version of every report, I'd say "You can switch between Spanish-specific tax reports by clicking..."